### ISSUE TITLE
fix todo about weak keypair generator

### ISSUE THREAD

**Issue by kmindi** [cryptoexamples/javascript-crypto-cryptoexamples#2]
https://github.com/cryptoexamples/javascript-crypto-cryptoexamples/blob/3d4904d2e934b7fd12f8471206ec59c311763f24/src/allinone/ExampleAsymmetricStringEncryption.js#L12

**Comment by Hirzelts** [comment]
I do not think that there is really an alternative. crypto has no pem-format-keypair generator. Besides, the keypair plugin im using here, is just a wrapper around forges (forge is the other lib) rsa-keypair generator. So it should be fine? It is a discussionworthy problem for the cryptoexamples, that crypto cant generate keypairs.

**Comment by kmindi** [comment]
Is it (the keypair generator) currently not secure or only inconvinient? 
translate the todo and make it understandable what the problem is (still in one line pls)

Does the plugin you use require another crypto library, your comment suggest that?
